src/main/java/com/kaleido/fetch/service/FetchService.java

```diff
@@ -145,6 +152,29 @@ public FetchService(KaptureClient<Media> mediaKaptureClient, KaptureClient<Batch
         return results;
     }
 
+    public List<String> getAllBarcodes(String activityName)
+    {
+    	List<String> barcodeArray =  new ArrayList<String>();
+       
+    	ResponseEntity<List<Experiment>> experimentList = experimentKaptureClient.findByFieldEquals("name", activityName);
+    	 
+         if (experimentList.getStatusCode().is2xxSuccessful() && experimentList.getBody() != null &&  experimentList.getBody().size() > 0) {
+        
+  	     var platmapsfromrepository =plateMapKaptureClient.findByFieldEquals("experimentId", experimentList.getBody().get(0).getId().toString());
```

Comment:
```suggestion
  	     var platmapsfromrepository = plateMapKaptureClient.findByFieldEquals("experimentId", experimentList.getBody().get(0).getId().toString());
```
Additionally "experimentId" should probably be abstracted out and made a constant somewhere so that we don't have magic strings